network/src/main/java/bisq/network/p2p/services/peergroup/PeerGroup.java

```diff
@@ -83,6 +83,7 @@ public PeerGroup(Node node, Config config, Set<Address> seedNodeAddresses, BanLi
 
     public void addReportedPeers(Set<Peer> peers) {
         reportedPeers.addAll(peers);
+        peerGroupService.addPersistedPeers(peers);
```

Comment:
when using the persistedPeersHandler interface it would be: persistedPeersHandler.addPersistedPeers(peers);

Reply:
Done